Musescore.config

```diff
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
```

Comment:
OK, I see, these are my commits ;-). Please be patient with me, as it is really not that easy to learn all these things ... 
Where do I find the number of the commit before 57d3883? Or will a naked git reset do that?